### PR TITLE
[OGM] 5.4.0.CR1 release

### PR DESCRIPTION
**Pull request by fax4ever**
http://staging.in.relation.to/2018/10/01/hibernate-ogm-5-4-CR1-released/

### CODE REVIEW

File: posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+:awestruct-layout: blog-post
+---
+
+http://hibernate.org/ogm/releases/5.4/#get-it[Hibernate OGM 5.4.0.CR1] has been released!
+
+Here's a list of the main changes:
+
+* We support Infinispan remote transaction over HotRod client
+* Java types `java.time.LocalDate`, `java.time.LocalDateTime` and `java.time.LocalTime` are natively supported as a field types
+* It is possible to configure MongoDB `ReadConcern` strategy
```
Comment (yrodiere) [inline posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc]: A link to our own documentation would help here, I think.

File: posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+* http://infinispan.org[Infinispan]
+** Remote: _org.hibernate.ogm:hibernate-ogm-infinispan-remote:5.4.0.Beta2_
+** Embedded: _org.hibernate.ogm:hibernate-ogm-infinispan-embedded:5.4.0.Beta2_
+* https://www.mongodb.com[MongoDB]: _org.hibernate.ogm:hibernate-ogm-mongodb:5.4.0.Beta2_
+* http://neo4j.com[Neo4j]: _org.hibernate.ogm:hibernate-ogm-neo4j:5.4.0.Beta2_
+** Infinispan Remote: _org.hibernate.ogm:hibernate-ogm-featurepack-infinispan-remote:5.4.0.Beta2_
+** Infinispan Embedded: _org.hibernate.ogm:hibernate-ogm-featurepack-infinispan-embedded:5.4.0.Beta2_
+** MongoDB: _org.hibernate.ogm:hibernate-ogm-featurepack-mongodb:5.4.0.Beta2_
+** Neo4j: _org.hibernate.ogm:hibernate-ogm-featurepack-neo4j:5.4.0.Beta2_
```
Comment (yrodiere) [inline posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc]: You should update these version numbers.

File: posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+* It is possible to configure MongoDB `ReadConcern` strategy
+
+More details available in the
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=31690[release notes].
+
+== Components upgrade
+
+* Hibernate ORM 5.3.4.Final
+* Hibernate Search 5.10.4.Final
+* Infinispan 5.3.3.Final
```
Comment (yrodiere) [inline posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc]: Looks like there's a typo here. The Infinispan version seems wrong.

**Review by yrodiere — COMMENTED** [review]
My two cents. Looks good apart from that.

File: posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc (hunk 1 above)
Comment (fax4ever) [reply posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc]: Good idea, thanks @yrodiere

File: posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc (hunk 3 above)
Comment (fax4ever) [reply posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc]: There definitely is. Thanks

File: posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc (hunk 1 above)
Comment (fax4ever) [reply posts/Fabio/2018-10-01-hibernate-ogm-5-4-CR1-released.adoc]: Unfortunately, the new property has not been documented yet.

**Comment by fax4ever** [comment]
Thanks @yrodiere

**Comment by fax4ever** [comment]
I applied here also some @Sanne's tips. Thank you @Sanne

**Pull request merged**